proteus/mprans/BoundaryConditions.py

```diff
@@ -244,8 +244,7 @@ def __cpp_MoveMesh_hy(self, x, t):
     def __cpp_MoveMesh_hz(self, x, t):
         return self.__cpp_MoveMesh_h(x, t)[2]
 
-    def setUnsteadyTwoPhaseVelocityInlet(self, wave, vert_axis=None, wind_speed=None,
-                                         smoothing=0., vof_air=1., vof_water=0.):
+    def setUnsteadyTwoPhaseVelocityInlet(self, wave,smoothing, vert_axis=None, wind_speed=None, vof_air=1., vof_water=0.):
```

Comment:
It would be nice to break the line here so it is limited to 69 characters in length like most of the rest of the file.